code/modules/mob/living/basic/vermin/mothroach.dm

```diff
@@ -5,9 +5,9 @@
 	icon_living = "mothroach"
 	icon_dead = "mothroach_dead"
 	held_state = "mothroach"
-	held_lh = 'icons/mob/animal_item_lh.dmi'
-	held_rh = 'icons/mob/animal_item_rh.dmi'
-	head_icon = 'icons/mob/animal_item_head.dmi'
+	held_lh = 'icons/mob/simple/held-pets/animal_item_lh.dmi'
```

Comment:
Should go in inhands

Reply:
done (as well as all the other comments)